extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/VertxOAuth2AuthenticationMechanism.java

```diff
@@ -24,16 +24,6 @@
 
     private volatile String authServerURI;
     private volatile OAuth2Auth auth;
-    private volatile OidcConfig config;
-
-    public OidcConfig getConfig() {
-        return config;
-    }
-
-    public VertxOAuth2AuthenticationMechanism setConfig(OidcConfig config) {
-        this.config = config;
-        return this;
-    }
```

Comment:
I don't think we should expose this and AFAICS it's not used. @michalszynkiewicz it's you who introduced that, can you confirm it's not used?

Reply:
Neither my memory nor git blame thinks I introduced that but I don't see `getConfig` used anywhere and the field is not used either.

Reply:
Oh sorry, my Eclipse history made me think you added this when working on the security interceptors :).

Yeah, I think it's safe to remove it. Will wait for @stuartwdouglas .